src/plugins/signoff/types.js

```diff
@@ -23,8 +23,8 @@ export type SourceInfo = {
   bid: string,
   cid: string,
   // Full info.
-  changesOnSource?: ChangesList,
-  changesOnPreview?: ChangesList,
+  changesOnSource?: ?ChangesList,
+  changesOnPreview?: ?ChangesList,
```

Comment:
Maybe a comment here explaining that they're absent if preview is disabled and/or only one is present depending on status. I think only one depending on status is reasonable but I think it could use a comment explaining why (maybe "If work-in-progress, show changes since the last review request. Preview cannot be signed directly in this status. Otherwise, show changes that are to be reviewed."). Alternately, is it possible that a user will want to see changes vs. destination as part of a complicated change/review? In that case, maybe we should fetch both.